en/sensor/pmw3901.md

```diff
@@ -30,7 +30,7 @@ An external rangefinder/distance sensor is *required* for the sensors that don't
 
 The range needed is dependent on the application:
 - Indoor flight: ≈4 meters
-- Outdoor flight: ≥10 metres (e.g. to support the position control in environments where GPS might have issues)
+- Outdoor flight: ≥10 meters (e.g. to support the position control in environments where GPS might have issues)
```

Comment:
FYI, metres is correct spelling for UK English, the language of this doc set. Pretty much only USA and Philippines use "meter".